src/policy/immix/immixspace.rs

```diff
@@ -51,6 +51,20 @@ impl<VM: VMBinding> SFT for ImmixSpace<VM> {
     fn name(&self) -> &str {
         self.get_name()
     }
+
+    #[inline(always)]
```

Comment:
Probably remove the `inline(always)` since we've been investigating this :laughing:

Reply:
You're right.  I'll remove that :)